addons/sms_twilio/tests/common.py

```diff
@@ -110,26 +129,58 @@ def _setup_sms_twilio(cls, company):
             "sms_provider": "twilio",
             "sms_twilio_account_sid": "AC12345678987654321234567898765432",
             "sms_twilio_auth_token": "grimgorironhide",
+            "sms_twilio_number_ids": [
+                (5, 0),
+                (0, 0, {
+                    "country_id": cls.env.ref("base.be").id,
+                    "number": "+32455998877",
+                    "sequence": 0,
+                }),
+                (0, 0, {
+                    "country_id": cls.env.ref("base.us").id,
+                    "number": "+15056998877",
+                    "sequence": 1,
+                }),
+            ],
         })
 
     @classmethod
-    def _update_mock(cls, mock_ok, mock_body, mock_number, mock_sms_uuid, mock_company):
-        cls.mock_ok = mock_ok
+    def _update_mock(cls, mock_error_type=None, mock_error_number_to_type=None,
```

Comment:
I probably shouldn't ask... but why are there "mock_" everywhere? They are all "_mock"! isn't the function name enough to grep and understand `cls.mock_body = body`?